websvc.jaxws21/external/jaxws-2.2.6-notice.txt

```diff
@@ -0,0 +1,10 @@
+Woodstox XML processor
+
+This product currently only contains code developed by authors
```

Comment:
I'm not sure what should we do with this notice. We are not redistributing code so this paragraph doesn't make sense.